src/libraries/System.Diagnostics.PerformanceCounter/src/System/Diagnostics/PerformanceCounterLib.cs

```diff
@@ -79,6 +79,18 @@ private static object InternalSyncObject
             }
         }
 
+        private static CultureInfo EnglishCulture
+        {
+            get
+            {
+                if (s_englishCulture is null)
+                {
+                    try { s_englishCulture = CultureInfo.GetCultureInfo("en"); } catch { s_englishCulture = CultureInfo.InvariantCulture; }
```

Comment:
Should it catch the specific exception type? I have no preference

Reply:
But I think our style is curlies on their own lines

Reply:
> Should it catch the specific exception type? I have no preference

Catching all exceptions will be safer as I think we shouldn't fail perf counters because of culture creation. 

> But I think our style is curlies on their own lines

I was trying to compact it to show it as one line assignment. I'll spread as you suggest.